src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/view.html.twig

```diff
@@ -32,55 +32,61 @@
 
 
 {% block content %}
-  <div class="row">
-
-    {% set displayAdminOrderTopHookContent = renderhook('displayAdminOrderTop', {'id_order': orderForViewing.id}) %}
+  <div id="order-view-page">
```

Comment:
Please fix indent for this file

Reply:
Fixed in https://github.com/PrestaShop/PrestaShop/pull/17617/commits/c5c493a09b80b93519121be4aeaa34e8f627d94e